[2024-03-16T03:04:37.828+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dbt_dag.my_first_dbt_model.test scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-03-16T03:04:37.836+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dbt_dag.my_first_dbt_model.test scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-03-16T03:04:37.836+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-16T03:04:37.855+0000] {taskinstance.py:2192} INFO - Executing <Task(DbtTestLocalOperator): my_first_dbt_model.test> on 2024-03-15 00:00:00+00:00
[2024-03-16T03:04:37.863+0000] {standard_task_runner.py:60} INFO - Started process 1668 to run task
[2024-03-16T03:04:37.866+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dbt_dag', 'my_first_dbt_model.test', 'scheduled__2024-03-15T00:00:00+00:00', '--job-id', '487', '--raw', '--subdir', 'DAGS_FOLDER/weather_dbt_dag.py', '--cfg-path', '/tmp/tmpoinknvd3']
[2024-03-16T03:04:37.867+0000] {standard_task_runner.py:88} INFO - Job 487: Subtask my_first_dbt_model.test
[2024-03-16T03:04:37.914+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dbt_dag.my_first_dbt_model.test scheduled__2024-03-15T00:00:00+00:00 [running]> on host a084c6a0d68a
[2024-03-16T03:04:38.003+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dbt_dag' AIRFLOW_CTX_TASK_ID='my_first_dbt_model.test' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T00:00:00+00:00'
[2024-03-16T03:04:38.004+0000] {local.py:202} INFO - Cloning project to writable temp directory /tmp/tmpx0wdrn35 from /opt/airflow/dbt/weather
[2024-03-16T03:04:38.013+0000] {base.py:83} INFO - Using connection ID 'my_***_conn' for task execution.
[2024-03-16T03:04:38.014+0000] {base.py:171} INFO - Using real values for profile weather
[2024-03-16T03:04:38.016+0000] {config.py:278} INFO - Creating temporary profiles.yml at /tmp/tmpzbo1znox/profiles.yml with the following contents:
weather:
    outputs:
        dev:
            dbname: ***
            host: mydb
            password: '{{ env_var(''COSMOS_CONN_POSTGRES_PASSWORD'') }}'
            port: 5432
            schema: dbt_gai
            type: ***
            user: ***
    target: dev

[2024-03-16T03:04:38.017+0000] {local.py:235} INFO - Trying to run the command:
 ['/home/***/.local/bin/dbt', 'test', '--models', 'my_first_dbt_model', '--profiles-dir', '/tmp/tmpzbo1znox', '--profile', 'weather', '--target', 'dev']
From /tmp/tmpx0wdrn35
[2024-03-16T03:04:38.018+0000] {local.py:236} INFO - Using environment variables keys: dict_keys(['AIRFLOW_CTX_DAG_OWNER', 'AIRFLOW_CTX_DAG_ID', 'AIRFLOW_CTX_TASK_ID', 'AIRFLOW_CTX_EXECUTION_DATE', 'AIRFLOW_CTX_TRY_NUMBER', 'AIRFLOW_CTX_DAG_RUN_ID', 'COSMOS_CONN_POSTGRES_PASSWORD'])
[2024-03-16T03:04:38.018+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-03-16T03:04:38.019+0000] {subprocess.py:73} INFO - Running command: ['/home/***/.local/bin/dbt', 'test', '--models', 'my_first_dbt_model', '--profiles-dir', '/tmp/tmpzbo1znox', '--profile', 'weather', '--target', 'dev']
[2024-03-16T03:04:38.035+0000] {subprocess.py:84} INFO - Command output:
[2024-03-16T03:04:40.639+0000] {subprocess.py:94} INFO - [0m03:04:40  Running with dbt=1.7.9
[2024-03-16T03:04:40.998+0000] {subprocess.py:94} INFO - [0m03:04:40  Registered adapter: ***=1.7.9
[2024-03-16T03:04:41.008+0000] {subprocess.py:94} INFO - [0m03:04:41  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2024-03-16T03:04:42.596+0000] {subprocess.py:94} INFO - [0m03:04:42  Found 5 models, 9 tests, 1 source, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2024-03-16T03:04:42.598+0000] {subprocess.py:94} INFO - [0m03:04:42
[2024-03-16T03:04:42.673+0000] {subprocess.py:94} INFO - [0m03:04:42  Concurrency: 1 threads (target='dev')
[2024-03-16T03:04:42.676+0000] {subprocess.py:94} INFO - [0m03:04:42
[2024-03-16T03:04:42.688+0000] {subprocess.py:94} INFO - [0m03:04:42  1 of 2 START test not_null_my_first_dbt_model_id ............................... [RUN]
[2024-03-16T03:04:42.751+0000] {subprocess.py:94} INFO - [0m03:04:42  1 of 2 PASS not_null_my_first_dbt_model_id ..................................... [[32mPASS[0m in 0.06s]
[2024-03-16T03:04:42.754+0000] {subprocess.py:94} INFO - [0m03:04:42  2 of 2 START test unique_my_first_dbt_model_id ................................. [RUN]
[2024-03-16T03:04:42.780+0000] {subprocess.py:94} INFO - [0m03:04:42  2 of 2 PASS unique_my_first_dbt_model_id ....................................... [[32mPASS[0m in 0.03s]
[2024-03-16T03:04:42.792+0000] {subprocess.py:94} INFO - [0m03:04:42
[2024-03-16T03:04:42.793+0000] {subprocess.py:94} INFO - [0m03:04:42  Finished running 2 tests in 0 hours 0 minutes and 0.19 seconds (0.19s).
[2024-03-16T03:04:42.803+0000] {subprocess.py:94} INFO - [0m03:04:42
[2024-03-16T03:04:42.804+0000] {subprocess.py:94} INFO - [0m03:04:42  [32mCompleted successfully[0m
[2024-03-16T03:04:42.805+0000] {subprocess.py:94} INFO - [0m03:04:42
[2024-03-16T03:04:42.806+0000] {subprocess.py:94} INFO - [0m03:04:42  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-03-16T03:04:43.987+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-03-16T03:04:43.996+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v11.json is above dbt-ol supported version 7. This might cause errors.
[2024-03-16T03:04:43.999+0000] {local.py:252} INFO - Inlets: [Dataset(uri='postgres://mydb:5432/postgres.dbt_gai.my_first_dbt_model', extra=None)]
[2024-03-16T03:04:44.000+0000] {local.py:253} INFO - Outlets: []
[2024-03-16T03:04:44.000+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-03-16T03:04:44.024+0000] {dag.py:3820} INFO - Setting next_dagrun for weather_dbt_dag to 2024-03-16T00:00:00+00:00, run_after=2024-03-17T00:00:00+00:00
[2024-03-16T03:04:44.052+0000] {local.py:371} INFO - [0m03:04:42  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-03-16T03:04:44.055+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1476 RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
[2024-03-16T03:04:44.078+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dbt_dag, task_id=my_first_dbt_model.test, execution_date=20240315T000000, start_date=20240316T030437, end_date=20240316T030444
[2024-03-16T03:04:44.112+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-16T03:04:44.139+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
